integration-tests/tests/awsdatapi.test.ts

```diff
@@ -444,7 +444,7 @@ test.serial('full join with alias', async (t) => {
 test.serial('select from alias', async (t) => {
 	const { db } = t.context;
 
-	const pgTable = pgTableCreator((name) => `prefixed_${name}`);
+	const pgTable = pgTableCreator((name: string) => `prefixed_${name}`);
```

Comment:
Why did you make this change?

Reply:
This was due to my editor complaining of no type existing for this. Since it's a string interpolation it made sense to be a string name, but if we should be expecting anything else on this variable, I'd be happy to amend it

Reply:
Updated